includes/rest-api/Controllers/Version2/class-wc-rest-orders-v2-controller.php

```diff
@@ -260,13 +260,28 @@ protected function get_formatted_item_data( $order ) {
 		$extra_fields      = array( 'meta_data', 'line_items', 'tax_lines', 'shipping_lines', 'fee_lines', 'coupon_lines', 'refunds' );
 		$format_decimal    = array( 'discount_total', 'discount_tax', 'shipping_total', 'shipping_tax', 'shipping_total', 'shipping_tax', 'cart_tax', 'total', 'total_tax' );
 		$format_date       = array( 'date_created', 'date_modified', 'date_completed', 'date_paid' );
+		// These fields are dependent on other fields.
+		$dependent_fields = array(
+			'date_created_gmt'   => 'date_created',
+			'date_modified_gmt'  => 'date_modified',
+			'date_completed_gmt' => 'date_completed',
+			'date_paid_gmt'      => 'date_paid',
+		);
+
 		$format_line_items = array( 'line_items', 'tax_lines', 'shipping_lines', 'fee_lines', 'coupon_lines' );
 
 		// Only fetch fields that we need.
-		$fields            = $this->get_fields_for_response( $this->request );
+		$fields = $this->get_fields_for_response( $this->request );
+		foreach ( $dependent_fields as $field_key => $dependency ) {
+			if ( in_array( $field_key, $fields ) && ! in_array( $dependency, $fields ) ) {
+				$fields[] = $dependency;
+			}
+		}
+
```

Comment:
I'm slightly confused here. Even without this code that you added, I still see the `date_created_gmt` in the response in master branch. Am I doing it wrong?

Reply:
As @vedanshujain explained to me in slack, this only becomes a problem if the field is explicitly specified using the `_fields` parameter.

Reply:
Turns out even `date_created_gmt` and `date_modified_gmt` are also dependencies because even though they are stored in post table, they are not part of $order->get_base_data(), and I think changing order class code is out of scope for this bug fix PR. So I have kept these changes as it is.